src/smallmap_gui.cpp

```diff
@@ -1685,6 +1732,14 @@ class SmallMapWindow : public Window {
 				if (_settings_client.sound.click_beep) SndPlayFx(SND_15_BEEP);
 				break;
 
+			case WID_SM_SHOW_IND_NAMES: // Toggle industry names
+				this->show_industry_names = !this->show_industry_names;
+				this->SetWidgetLoweredState(WID_SM_SHOW_IND_NAMES, this->show_industry_names);
+
+				this->SetDirty();
+				if (_settings_client.sound.click_beep) SndPlayFx(SND_15_BEEP);
+				break;
+
```

Comment:
Follows the existing `WID_SM_TOGGLETOWNNAME`:

https://github.com/OpenTTD/OpenTTD/blob/e35680d0cbf8fe8e7c4b55729a45e9b0257f09fa/src/smallmap_gui.cpp#L1727-L1733